### DISCUSSION TITLE
要不要在教程中加入“经典”异常？

### DISCUSSION THREAD

**Discussion by EwTE-H** [Physics365/991CN-X-CW-Decompilation#48] [Ideas]
看教程的目录，里面好像主要都是代码和一些原理，很少有经典异常的例子，连b站常见的太阳能爆机都没有。我个人认为，教程的四分之一到三分之一完全不涉及汇编，是比较合适的。

**Comment by Physics365** [comment]
是的，这些通常是引人兴趣的部分，且原理较为复杂（反正我说不清），可以单独放一块
